torch/_inductor/fx_passes/quantization.py

```diff
@@ -1162,16 +1264,26 @@ def _generate_qlinear_weight_prepack_patterns():
 
 @functools.lru_cache(None)
 def _register_quantization_weight_pack_pass():
-    # Step 1: Dequant promotion
-    _register_dequant_promotion_pass(
-        dequantize_per_tensor_activation_pattern, pass_number=0
-    )  # pass_number=0 to run before weight prepack
-
-    # Step 2: QConv weight prepack
-    weight_prepack_patterns = _generate_qconv_weight_prepack_patterns()
-    for weight_prepack_pattern in weight_prepack_patterns:
-        # Register to pass_number 1, so we can do dequant promotion in pass_number 0.
-        _register_qconv_weight_prepack_pass(weight_prepack_pattern, pass_number=1)
+    for dtype in [torch.float32, torch.bfloat16]:
+        # Step 1: Dequant promotion for int8-mixed-fp32/bf16
```

Comment:
what does dequant promotion mean? can you write down the patterns or transformation?

Reply:
Thanks for the suggestion. Add the patterns and description for this pass.